tests/ovm_test.go

```diff
@@ -0,0 +1,92 @@
+package tests
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/core/vm/runtime"
+)
+
+var KEY = common.FromHex("0102030000000000000000000000000000000000000000000000000000000000")
+var VALUE = common.FromHex("0405060000000000000000000000000000000000000000000000000000000000")
+
+func mstoreBytes(bytes []byte, offset int) []byte {
+	output := make([]byte, len(bytes)*5)
+	for i, b := range bytes {
+		output[i*5] = byte(vm.PUSH1)
+		output[i*5+1] = b
+		output[i*5+2] = byte(vm.PUSH1)
+		output[i*5+3] = byte(offset + i)
+		output[i*5+4] = byte(vm.MSTORE8)
+	}
+	return output
+}
+
+func call(addr []byte, value uint, inOffset uint, inSize uint, retOffset uint, retSize uint) []byte {
+	output := []byte{
+		byte(vm.PUSH1), 0,
+		byte(vm.PUSH1), 0,
+		byte(vm.PUSH1), byte(retSize),
+		byte(vm.PUSH1), byte(retOffset),
+		byte(vm.PUSH1), byte(inSize),
+		byte(vm.PUSH1), byte(inOffset),
+		byte(vm.PUSH1), byte(value),
+	}
+	output = append(output, []byte{
+		byte(vm.PUSH20)}...)
+	output = append(output, addr...)
+	output = append(output, []byte{
+		byte(vm.GAS),
+		byte(vm.CALL),
+	}...)
+	return output
+}
+
+func TestOvm(t *testing.T) {
```

Comment:
FWIW, in order to set context properly, `ovmSSTORE` and `ovmSLOAD` are best tested with an involved setup where the tx does an `ovmCALL` to another contract C (though the OVM), and then C calls `ovmSSTORE` so that the storage is associated with C within `ExecutionManager`. [this](https://github.com/op-optimism/optimistic-rollup/blob/master/packages/ovm/test/contracts/execution-manager.storage-opcodes.spec.ts) has some examples.